content/docs/rosa/gpu/_index.md

```diff
@@ -24,9 +24,7 @@ Enter the `oc login` command, username, and password from the output of the prev
 
 Example login:
 ```bash
-oc login https://api.cluster_name.t6k4.i1.organization.org:6443 \
-> --username cluster-admin \
-> --password FWGYL-2mkJI-3ZTTZ-rINns 
+oc login https://api.cluster_name.t6k4.i1.organization.org:6443 --username cluster-admin
```

Comment:
IMHO I think it's okay to leave some random username and password in there, it's just an example anyway:)

Reply:
@dsari-rh I was thinking also that, but my security past background was chasing me and I didn't wanted to put a clear password that could lead the possible customer to do that in their clusters (and also because sometimes the security bots does warning these things). We can stick with the clear password this if it's ok for you :)

Reply:
lol okay, or maybe let's do `mypa55w0rd` like we do in many of our docs ;)